zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/FollowerZooKeeperServer.java

```diff
@@ -31,8 +31,10 @@
 import org.apache.zookeeper.server.RequestProcessor;
 import org.apache.zookeeper.server.ServerMetrics;
 import org.apache.zookeeper.server.SyncRequestProcessor;
+import org.apache.zookeeper.server.TxnLogEntry;
```

Comment:
I don't see where this is defined. Maybe in a new file that's forgot to be included as part of this PR?

Reply:
This one should not be here.